Code/Tools/FBuild/FBuildCore/FBuildOptions.cpp

```diff
@@ -178,6 +178,26 @@ FBuildOptions::OptionsResult FBuildOptions::ProcessCommandLine( int argc, char *
                 m_Args += argv[ sizeIndex ];
                 continue;
             }
+            else if (thisArg == "-workerlimit")
+            {
+                const int sizeIndex = (i + 1);
+                PRAGMA_DISABLE_PUSH_MSVC(4996) // This function or variable may be unsafe...
+                PRAGMA_DISABLE_PUSH_CLANG_WINDOWS("-Wdeprecated-declarations") // 'sscanf' is deprecated: This function or variable may be unsafe...
+                if ((sizeIndex >= argc) ||
+                     (sscanf(argv[sizeIndex], "%i", &m_workerlimit) != 1))
+                PRAGMA_DISABLE_POP_CLANG_WINDOWS // -Wdeprecated-declarations
+                PRAGMA_DISABLE_POP_MSVC // 4996
+                {
+                    OUTPUT("FBuild: Error: Missing or bad <worker> for '-workerlimit' argument\n");
```

Comment:
This error message refers to "bad <worker>" which is not the name of the parameter (number_worker)

Reply:
Could I modify the output?
`OUTPUT("FBuild: Error: Missing <number_worker> for '-workerlimit' argument\n")`